apps/web-giddh/src/app/subscription/subscription-list/subscription-list.component.ts

```diff
@@ -0,0 +1,747 @@
+        if (searchedFieldName === 'companyName') {
+            if (this.subscriptionListForm?.controls['companyName'].value !== null && this.subscriptionListForm?.controls['companyName'].value !== '') {
+                return;
+            }
+        } else if (searchedFieldName === 'Billing Account') {
+            if (this.subscriptionListForm?.controls['billingAccountName'].value !== null && this.subscriptionListForm?.controls['billingAccountName'].value !== '') {
+                return;
+            }
+        } else if (searchedFieldName === 'Subscriber') {
+            if (this.subscriptionListForm?.controls['subscriberName'].value !== null && this.subscriptionListForm?.controls['subscriberName'].value !== '') {
+                return;
+            }
+        } else if (searchedFieldName === 'Country') {
+            if (this.subscriptionListForm?.controls['countryName'].value !== null && this.subscriptionListForm?.controls['countryName'].value !== '') {
+                return;
+            }
+        } else if (searchedFieldName === 'Plan Name') {
+            if (this.subscriptionListForm?.controls['planName'].value !== null && this.subscriptionListForm?.controls['planName'].value !== '') {
+                return;
+            }
+        } else if (searchedFieldName === 'Status') {
+            if (this.subscriptionListForm?.controls['status'].value !== null && this.subscriptionListForm?.controls['status'].value !== '') {
+                return;
+            }
+        } else if (searchedFieldName === 'Monthly/Yearly') {
+            if (this.subscriptionListForm?.controls['duration'].value !== null && this.subscriptionListForm?.controls['duration'].value !== '') {
+                return;
+            }
+        }
+
+        if (this.generalService.childOf(event?.target, element)) {
+            return;
+        } else {
+            if (searchedFieldName === 'Company') {
+                this.showName = false;
+            } else if (searchedFieldName === 'Billing Account') {
+                this.showBillingAccount = false;
+            } else if (searchedFieldName === 'Subscriber') {
+                this.showSubscriber = false;
+            } else if (searchedFieldName === 'Country') {
+                this.showCountry = false;
+            } else if (searchedFieldName === 'Plan Name') {
+                this.showPlanSubName = false;
+            } else if (searchedFieldName === 'Monthly/Yearly') {
+                this.showMonthlyYearly = false;
+            } else if (searchedFieldName === 'Status') {
+                this.showStatus = false;
+            }
+        }
+    }
+
+    /**
+     * This will be use for toggle search field
+     *
+     * @param {string} fieldName
+     * @param {*} el
+     * @memberof SubscriptionComponent
+     */
+    public toggleSearch(fieldName: string): void {
+        if (fieldName === 'Company') {
```

Comment:
this won't work since we are sending translated data here so in other languages this won't work